controllers/validator.go

```diff
@@ -62,19 +70,15 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 			}
 		}
 
-		// Also check if the VSM plugin is specified or not
-		VSMPluginPresent := false
-		for _, plugin := range dpa.Spec.Configuration.Velero.DefaultPlugins {
-			if plugin == oadpv1alpha1.DefaultPluginVSM {
-				VSMPluginPresent = true
-			}
-		}
-
 		if !VSMPluginPresent {
 			return false, errors.New("datamover feature is enabled, need to specify vsm as a default plugin in DataProtectionApplication CR")
 		}
 	}
 
+	if !r.checkIfDataMoverIsEnabled(&dpa) && VSMPluginPresent {
+		return false, errors.New("datamover feature is disabled, need to remove vsm as a default plugin in DataProtectionApplication CR")
```

Comment:
could possibly shorten this to something like "datamover is disabled-remove vsm as a default plugin"